### DISCUSSION TITLE
Recently Played Fix

### DISCUSSION THREAD

**Discussion by Jfroi** [SpotX-Official/SpotX#151] [Ideas] [answered]
it would so really nice if private session didn't show recently played songs at home, or if recently played is fully removed

**Comment by amd64fox** [comment]
I can just remove this section for your Spotify instance, send your `xpui.spa` here.

But for Spotx in general I don't think that's a good idea. At least it is difficult to implement in these conditions, but it is easy to implement in [Spicetify](https://github.com/spicetify/spicetify-cli) conditions.

**Comment by Jfroi** [comment, marked as answer]
hmm yeah, may know what xpui.spa is?

**Reply by amd64fox** [reply]
- <kbd>Win</kbd> + <kbd>R</kbd>
- `%appdata%\Spotify\Apps`
- make a copy of `xpui.spa` and rename it to `xpui.zip`
-  send `xpui.zip`

**Reply by Jfroi** [reply]
[xpui.zip](https://github.com/amd64fox/SpotX/files/9594004/xpui.zip)

**Reply by amd64fox** [reply]
ok, just a moment

**Reply by amd64fox** [reply]
turned off the `recently played` section
[xpui.zip](https://github.com/amd64fox/SpotX/files/9594954/xpui.zip)

if the client is updated then everything will be lost

**Comment by Jfroi** [comment]
ok thank you very much
